README.md

```diff
@@ -85,9 +111,9 @@ new_value = client.get('conjur/my/variable')
 print("Variable value is:", new_value.decode('utf-8'))
 ```
 
-#### With login Id and API key
+#### With login ID and API key
 
-Write the code same as in the first example but create the client with the following arguments:
+Write the code same as in the first example but initialize the Client with the following arguments:
```

Comment:
that is so american :)

I would say 
Write the code as in the first example
or 
Write the code in the same way as in the first example

but the american was is fine too, i think